scrapyd/sqlite.py

```diff
@@ -144,6 +147,24 @@ def clear(self):
         self.conn.commit()
         self.update_project_priority_map()
 
+    def ensure_insert_time_column(self):
+        q = "SELECT insert_time FROM %s LIMIT 1" % self.table
```

Comment:
There's also:
```sql
SELECT sql FROM sqlite_master WHERE type='table' AND name='spider_queue';
-- ⇒ CREATE TABLE spider_queue (id integer primary key, priority real key, message blob)
```
but is it any better?

Reply:
I didn't know about 'sqlite_master' before.
The ensure_insert_time_column method is updated in [the 7th commit](https://github.com/scrapy/scrapyd/pull/344/commits/0343475f6e95043c710134823d29d170d56f2e96).